vendor/github.com/nsf/termbox-go/syscalls.go

```diff
@@ -0,0 +1,39 @@
+// +build ignore
```

Comment:
wat?

Reply:
I don't think this is too important. If you `git grep \"C\", you'll see many other vendored packaged do this as well. There might be some way of telling dep to prune cgo source.